src/services/mail.ts

```diff
@@ -0,0 +1,70 @@
+import nodemailer, { SentMessageInfo } from 'nodemailer'
+
+const MAX_TIME_RETRY_SEND_MAIL = process.env.MAX_TIME_RETRY_SEND_MAIL || 3
+export async function sendMail(
```

Comment:
Use [ES6 arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) syntax:

```suggestion
export const sendMail = async (
```